plugins/github/github.py

```diff
@@ -0,0 +1,42 @@
+import requests
+import re
+import os
+
+import github3
+
+from errbot import BotPlugin, botcmd
+
+GH = github3.login(token=os.environ.get("GITHUB_TOKEN"))
+if not GH:
+    raise ValueError("Environment variable GITHUB_TOKEN not set")
+
+class GitHub(BotPlugin):
+
+    @botcmd(split_args_with=" ")
+    def gh_assign(self, msg, args):
+        """
+        Assign a given issue to the user who issues the cmd.
+        """
+        
+        issue_link = args[0]
```

Comment:
Give a nice error maybe?
Because it can be confusing why someone was not assigned